### DISCUSSION TITLE
Trouble with controller mapping

### DISCUSSION THREAD

**Discussion by shaunx777** [AntiMicroX/antimicrox#1102] [Q&A] [answered]
OS: Linux Mint 22
Controller: BDA PS3 Airflo wired controller

I have a generic Playstation controller that I can't get games to recognise. Many of the buttons are incorrectly mapped, and I'm trying to use AntiMicroX to solve this. I was able to remap the buttons and joysticks in the app (i tried flatpak and .deb package) and they looked fine within the app but outside of it there was no change. I did restart my computer and that didn't change anything either. I'm assuming I'm doing something wrong or missing a step although I did come across my issue on reddit which resulted in the same problem I'm stuck at.

**Comment by pktiuk** [comment, marked as answer]
AntiMicroX does not change the layout of the buttons recognized by system.     
AntiMicroX is used for mapping gamepad buttons to keyboard buttons (and some variations of buttons/commands etc.) and nothing more.

**Reply by shaunx777** [reply]
Ah I see, thanks for letting me know. From what I read online it seemed like this tool could also be used to map funky controllers.

**Reply by pktiuk** [reply]
Yes, it could be used for mapping funky controllers to keyboard ;)
